src/Components/ServiceScene/Breakdowns/FieldsBreakdownScene.tsx

```diff
@@ -283,7 +283,7 @@ export class FieldsBreakdownScene extends SceneObjectBase<FieldsBreakdownSceneSt
         continue;
       }
 
-      const query = buildDataQuery(getExpr(optionValue), {
+      const query = buildDataQuery(getExpr(optionValue, false), {
```

Comment:
I'd suggest to create a new function like `getDrillDownExpr` instead of adding a boolean, as it helps a lot when reading the code.